src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url('/src/images/star-active.svg');
```

Comment:
The URL in the background-image property should be consistent. Consider using double quotes for the URL to match the style used in line 5.